frontend/src/Components/Post/Post.js

```diff
@@ -170,11 +170,11 @@ export default function Post({ handleDeletePost, post_initializer }) {
         } else {
           toast.warn(JSON.stringify(res.data.message));
           setModalText(
-            "Original Author: " +
-              details.author_name +
-              " (" +
-              details.author_email +
-              ")"
+            <A href={`/user/${details.author_id}`}>
+              "Original Author:
+              {details.author_name}({details.author_email}
+              )"{" "}
+            </A>
```

Comment:
Not sure about the exact tailwind property but this should at least make it look like a link when hovered:

```suggestion
            Original Author: 
            <A href={`/user/${details.author_id}`} className="hover: text-color-700">
              {details.author_name} ({details.author_email}
              ){" "}
            </A>
```

Reply:
`className="hover:text-blue-700"` would work